src/composables/latestTransactionList.ts

```diff
@@ -25,17 +38,40 @@ const { onNetworkChange } = createNetworkWatcher();
 export function useLatestTransactionList({ store }: IDefaultComposableOptions) {
   const { accounts } = useAccounts({ store });
   const { balancesTotal } = useBalances({ store });
+  const { nodeNetworkId } = useSdk({ store });
 
   const tokens = computed(() => store.state.fungibleTokens.tokens);
 
+  const localPendingTransactionsList = computed(
+    (): IPendingTransaction[] => (store.state.transactions as ITransactionsState)
+      .pending[nodeNetworkId.value!] || [],
+  );
+
+  function buildLatestTransactions() {
+    pendingTransactions.value = pendingTransactions.value.filter(({ hash }) => (
+      !fetchedTransactions.value.some((transaction) => transaction.hash === hash)
+    )) as IPendingTransaction[];
+
+    const transactions: ITransaction[] = [
+      ...pendingTransactions.value as ITransaction[],
```

Comment:
`pendingTransactions` has the type of `IPendingTransaction` - I suggest to avoid forcing the type here as this is not true.